apps/storybook/src/Zoom.stories.tsx

```diff
@@ -10,13 +10,15 @@ interface TemplateProps {
 }
 
 const Template: Story<TemplateProps> = (args) => {
+  const { disabled, modifierKey } = args;
```

Comment:
For convenience, I choose the keep the stories unchanged (only one `ModifierKey` can be selected with the radio button).

But the API of all interactions components (`Pan`, `Zoom`, `SelectToZoom` ...) changed to accept several modifier keys. This can be up to discussion.

Reply:
Good idea :+1: